scripts/site/_site/doc/app/app.component.ts

```diff
@@ -85,9 +86,20 @@ export class AppComponent implements OnInit {
     if (!this.platform.isBrowser) {
       return;
     }
+    if (theme === 'compact') {
+      this.nzConfigService.set('select', {
+        nzOptionHeightPx: 28
+      });
+    } else {
+      this.nzConfigService.set('select', {
+        nzOptionHeightPx: 32
+      });
```

Comment:
Hi @EnochGao 
I believe this file is for the document site; setting global configs here works on your localhost, because you are developing in the document site, but other developers are not.

In here you have access to the theme type, because the flow button that you can see at the right bottom corner of NG-ZORRO site sets it; in our components we cannot identify what theme developers are using.

Reply:
This issue has been previously discussed in #7304, which confirms the same problem. @cipchk's comment agrees with my earlier remark that theme type detection is not feasible in components.

Reply:
> Hi @EnochGao I believe this file is for the document site; setting global configs here works on your localhost, because you are developing in the document site, but other developers are not.
> 
> In here you have access to the theme type, because the flow button that you can see at the right bottom corner of NG-ZORRO site sets it; in our components we cannot identify what theme developers are using.

hi,Here's just fixing the site.The source code made no modifications, but instead added configuration properties to enable users to make global unified changes in their projects.

Reply:
> This issue has been previously discussed in #7304, which confirms the same problem. @cipchk's comment agrees with my earlier remark that theme type detection is not feasible in components.

Because some projects switch themes dynamically, it is impractical for each component to change on its own

Reply:
Oh Ok then; just please update the documentations so others would know that `nzVirtualItemSize` and `nzOptionHeightPx` are global configs.

Reply:
> Oh Ok then; just please update the documentations so others would know that and are global configs.`nzVirtualItemSize``nzOptionHeightPx`

ok，My mistake. I'll add it right away

Reply:
And I suggest you follow the issue template; otherwise it might get closed. 
The github checks not getting started are probably caused by this as well; i remember those issues not following the template would get closed automatically.

Reply:
> And I suggest you follow the issue template; otherwise it might get closed. The github checks not getting started are probably caused by this as well; i remember those issues not following the template used to get closed automatically.

thanks